playbook/app/pb_kits/playbook/pb_advanced_table/SubKits/TableBody.tsx

```diff
@@ -54,93 +57,250 @@ export const TableBody = ({
     className
   )
 
-  const columnPinning = table.getState().columnPinning;
+  const columnPinning = table.getState().columnPinning || { left: [] };
+
+  const style: React.CSSProperties = virtualizer ? {
+    height: `${virtualizer.getTotalSize()}px`, // tells scrollbar how big the table is
+    position: 'relative', // needed for absolute positioning of rows
+    width: '100%',
+  } : {};
+
+  // Additional css for virtualized table
+  const virtualizedCSS = `
+    .virtualized-table-row {
+      display: table !important;
+      table-layout: fixed !important;
+      width: 100% !important;
+    }
+
+    .virtualized-table-row td {
+      display: table-cell !important;
+    }
+
+    /* Ensure the first column has proper width */
+    .virtualized-table-row td:first-child {
+      width: auto !important;
+    }
+  `;
```

Comment:
Hey @skduncan any particular reason we have to use a style tag and can't use a classname and SCSS? Classnames are always better if possible!